main/lsp/AutocorrectFlusher.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SORBET_AUTOCORRECT_FLUSHER_H
+#define SORBET_AUTOCORRECT_FLUSHER_H
+
+#include "core/ErrorFlusher.h"
+
+namespace sorbet::realmain::lsp {
+
+class AutocorrectFlusher : public core::ErrorFlusher {
```

Comment:
This flusher is only used within `LSPTypechecker.cc`, correct? If so, we can define it there and not have a header file at all.